libraries/apollo-ast/src/commonMain/kotlin/com/apollographql/apollo3/ast/gqldirective.kt

```diff
@@ -135,13 +135,18 @@ private fun GQLValue?.toCatchTo(): CatchTo {
   }
 }
 
+private fun GQLDirective.isDefinedAndMatchesOriginalName(schema: Schema, originalName: String): Boolean {
+  return schema.directiveDefinitions.get(name) != null && schema.originalDirectiveName(name) == originalName
```

Comment:
Apologies for nitpicking this but I'd rather have the codegen crash than silently ignore missing directive definitions. 
All directive usage of `@catch` must be validated at that point so if we don't have a directive definition, it's a programming error that I'd rather catch (no pun intended) early

Reply:
Not sure why I had the impression it would still NPE without this, but it's not the case - just removed it 👍